src/plugins/vis_type_timeseries/server/lib/vis_data/response_processors/series/drop_last_bucket.js

```diff
@@ -14,7 +14,7 @@ export function dropLastBucket(resp, panel, series) {
     const shouldDropLastBucket = isLastValueTimerangeMode(panel, series);
 
     if (shouldDropLastBucket) {
-      const seriesDropLastBucket = get(series, 'override_drop_last_bucket', 1);
+      const seriesDropLastBucket = get(series, 'override_drop_last_bucket', 0);
```

Comment:
I think the following code works wrong, `override_drop_last_bucket` should has a priority to`drop_last_bucket`
```
const seriesDropLastBucket = get(series, 'override_drop_last_bucket', 0);
const dropLastBucket = get(panel, 'drop_last_bucket', seriesDropLastBucket);
```

@DziyanaDzeraviankina @stratoula think we can that in scope of that PR